pkg/virtctl/vnc/vnc.go

```diff
@@ -49,7 +49,7 @@ const (
 	//#### Tiger VNC ####
 	//# https://github.com/TigerVNC/tigervnc/releases
 	// Compatible with multiple Tiger VNC versions
-	MACOS_TIGER_VNC_PATTERN = `/Applications/TigerVNC Viewer*.app/Contents/MacOS/TigerVNC Viewer`
+	MACOS_TIGER_VNC_PATTERN = `/Applications/TigerVNC Viewer *.app/Contents/MacOS/TigerVNC Viewer`
```

Comment:
Is this change required for adding the flags? Does it fix a bug?

Reply:
I'd a little more strict on the naming convention of the TigerVNC binary. It's not required though.